src/Libraries/OrchardCore.Commerce.MoneyDataType/Currency.extra.cs

```diff
@@ -6,6 +6,8 @@
 
 namespace OrchardCore.Commerce.MoneyDataType;
 
+// Unrecognized (but valid) entries in a spell checking report should be added to
+// ".github/actions/spelling/allow/currency.txt".
```

Comment:
According to this, wouldn't adding `BitCoin` as an entry to currency.txt prevent needing to ignore lines in `KnownCurrencyTable` (ln. 64) and `IPayment` (ln. 12)? In the case of the former, I assume it might be the `₿` sign that triggers it, but for the latter, I don't see anything else that's problematic.

Reply:
Oooops, I actually removed currency.txt since then! The `₿` sign doesn't matter, check-spelling by default only cares about English alphabet characters. I'll add `Bitcoin` to occ.txt, because it's reasonable to assume that it will have more occurrences in the future other than the current 2.